contents/docs/self-host/deploy/troubleshooting.md

```diff
@@ -95,36 +95,33 @@ TooManyConnections: too many connections
 
 2. Get the logs for that pod using the name from the previous step:
    
-    ```shell
+    ```bash
     kubectl logs posthog-plugins-54f324b649-66afm -n posthog
     ```
 ## How do I connect to Postgres?
     
-> **Tip:** Find out your pod names with `kubectl get pods -n posthog`
-
 1. Find out your Postgres password from the web pod:
 
-    ```shell
-    kubectl exec -n posthog -it your-posthog-web-pod \
-    -- sh -c 'echo password:$POSTHOG_DB_PASSWORD'
+    ```bash
+    # First we need to determine the name of the web pod – see "How do I see logs for a pod?" for more on this
+    POSTHOG_WEB_POD_NAME=$(kubectl get pods -n posthog | grep -- '-web-' | awk '{print $1}')
+    # Then we can get the password from the pod's environment variables
+    kubectl exec -n posthog -it $POSTHOG_WEB_POD_NAME -- sh -c 'echo The Postgres password is: $POSTHOG_DB_PASSWORD'
     ```
 
 2. Connect to your Postgres pod:
 
-    ```shell
-    # Replace posthog-posthog-postgresql-0 with your pod's name if different
-    kubectl exec -n posthog -it posthog-posthog-postgresql-0  -- /bin/bash
-    ```
-
-3. Connect to the `posthog` DB:
```

Comment:
let's keep this step separate. Sometimes we'll want to connect to the pod to check storage or something else. Plus If we always do it 1. connect to the pod 2. do something on the pod, people will learn how to do (1) as it's always the same and (2) could be thought of as if you were on your machine terminal, what would you do.

Reply:
Makes senes, reverted to them being separate steps.